src/components/Buttons/Button.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import { getIcon } from '@components/Buttons/utils';
+
+import BaseButton from './BaseButton';
+import BaseButtonText from './BaseButtonText';
+import { getButtonProperties, getIconProperties } from './utils';
+
+import type { ButtonProps } from './types';
+/**
+ * Primary buttons have a colored background.
+ *
+ *  Use these for the main action in a section or screen. You should display only one primary button at a time.
+ *
+ */
+function Button({
+  accessibilityLabel = '',
+  disabled = false,
+  fill = false,
```

Comment:
Optional props already have false value.
```suggestion
  disabled,
  fill,
```

Reply:
Fixed https://github.com/magnetis/astro-native/pull/265/commits/01ad556544c0cc2de509d82ac90cf681a2306b88